cri/v1alpha2/cri_utils.go

```diff
@@ -814,6 +814,20 @@ func filterCRIContainers(containers []*runtime.Container, filter *runtime.Contai
 	return filtered
 }
 
+// matchLabelSelector returns true if labels cover selector.
+func matchLabelSelector(selector, labels map[string]string) bool {
```

Comment:
This function is uncoupled with `runtime`, maybe create a directory like `utils` under the cri directory for more public utility methods and move this funtion over. WDYT?

Reply:
Yes, we could do it in another PR and move all the public utility methods to there.

Reply:
@YaoZengzeng please put it in this PR. 😄 can you do this?